js/google_ima.js

```diff
@@ -1117,6 +1123,10 @@ require("../html5-common/js/utils/utils.js");
 
           google.ima.settings.setDisableCustomPlaybackForIOS10Plus(this.enableIosSkippableAds);
 
+          if (this.maxRedirects && typeof (this.maxRedirects) === 'number' && this.maxRedirects > 0) {
```

Comment:
Not sure if needed, but do we need to handle numerical string values gracefully or would it be ok to ignore them in this case?